client-management-service-impl/src/main/java/io/mosip/esignet/services/ClientManagementServiceImpl.java

```diff
@@ -203,20 +221,7 @@ public io.mosip.esignet.core.dto.ClientDetail getClientDetails(String clientId)
     @CacheEvict(value = Constants.CLIENT_DETAIL_CACHE, key = "#clientDetailCreateRequestV2.getClientId()")
     @Override
     public ClientDetailResponse createOAuthClient(ClientDetailCreateRequestV2 clientDetailCreateRequestV2) throws EsignetException {
-        Optional<ClientDetail> result = clientDetailRepository.findById(clientDetailCreateRequestV2.getClientId());
-        if (result.isPresent()) {
-            log.error("Duplicate Client Id : {}", ErrorConstants.DUPLICATE_CLIENT_ID);
-            throw new EsignetException(ErrorConstants.DUPLICATE_CLIENT_ID);
-        }
-
-        ClientDetail clientDetail = buildClientDetailEntity(clientDetailCreateRequestV2);
-
-        String clientName = getClientNameLanguageMapAsJsonString(
-                clientDetailCreateRequestV2.getClientNameLangMap(),
-                clientDetailCreateRequestV2.getClientName()
-        );
-        clientDetail.setName(clientName);
-
+        ClientDetail clientDetail = createClientHelper(clientDetailCreateRequestV2);
```

Comment:
createClientHelper and updateClientHelper are incorrect method names, method names should be a verb ... can you change the name to createClient and updateClient or any other valid action word